src/main.rs

```diff
@@ -390,17 +390,21 @@ fn run() -> Result<(), failure::Error> {
     } else if let Some(matches) = matches.subcommand_matches("generate") {
         let name = matches.value_of("name").unwrap_or("worker");
         let site = matches.is_present("site");
+        let template = matches.value_of("template");
         let mut target_type = None;
 
         let template = if site {
+            if template.is_some() {
+                failure::bail!("You cannot specify a template when generating a Workers Site. If you'd like to use the default site boilerplate, run wrangler generate --site. If you'd like to use another site boilerplate, omit the --site when running wrangler generate.")
```

Comment:
lol i'm still on this, something seems off here

"**You cannot specify a template when generating a Workers Site.** If you'd like to use the default site boilerplate, run wrangler generate --site. If you'd like to use another site boilerplate, omit the --site when running wrangler generate."

I think if we change the bolded part to "You've passed both `--site` and a template to `wrangler generate`- however you can only pass one or the other." 

what do you think?